packages/cadl-ranch-specs/http/payload/media-type/main.tsp

```diff
@@ -0,0 +1,56 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+import "@azure-tools/typespec-client-generator-core";
+
+using TypeSpec.Http;
+using Azure.ClientGenerator.Core;
+
+/**
+ * Test the payload with different media types and different types of the payload itself.
+ */
+@supportedBy("dpg")
+@scenarioService("/payload/media-type")
+namespace Payload.MediaType;
+
+// TODO: this file currently only contains string as body, we will add bytes body, model body soon.
+
+@scenario
+@scenarioDoc("""
+  When the request body and response body are both strings, the different media types would lead to different payloads.
+  """)
+@operationGroup
+namespace StringBody {
+  /**
+   * Expected request body is a string '{"Cat": "Meow"}'.
+   */
+  @post
+  @route("/sendStringAsText")
+  op sendStringAsText(@header contentType: "text/plain", @body text: string): OkResponse;
```

Comment:
the name can just be `sendAsText` because `StringBody` will already be part of the title